packages/circuits/email-verifier.circom

```diff
@@ -68,33 +69,34 @@ template EmailVerifier(max_header_bytes, max_body_bytes, n, k) {
     signal shifted_bh_out[LEN_SHA_B64] <== VarShiftLeft(max_header_bytes, LEN_SHA_B64)(bh_reveal, body_hash_idx);
     // log(body_hash_regex.out);
 
+    if (ignore_body_hash_check != 1) {
```

Comment:
Note: This diff look messed up but the only change is move the whole block below under this if condition.